agent/api/task/task.go

```diff
@@ -1016,28 +1017,21 @@ func (task *Task) shouldOverridePidMode(container *apicontainer.Container, docke
 
 	// no need to override the PID mode. This means the container's PID namespace
 	// will be private
-	if task.getPidMode() == "" || strings.EqualFold(task.getPidMode(), "none") {
+	if task.getPIDMode() == "" || strings.EqualFold(task.getPIDMode(), "none") {
 		return false, ""
 	}
 
-	if strings.EqualFold(task.getPidMode(), "host") {
+	if strings.EqualFold(task.getPIDMode(), "host") {
 		return true, "host"
-	} else if strings.EqualFold(task.getPidMode(), "task") {
-		//need to find the ContainerNamespacePause and assign
-		pidPauseContName := ""
-		for _, cont := range task.Containers {
-			if cont.Type == apicontainer.ContainerNamespacePause {
-				pidPauseContName = cont.Name
-				break
-			}
-		}
-		if pidPauseContName == "" {
-			seelog.Critical("Namespace Pause Container required, but not found in the task: %s", task.String())
-			return false, "" // cannot override the Pid without the Pause container
+	} else if strings.EqualFold(task.getPIDMode(), "task") {
```

Comment:
Any particular reason using `strings.EqualFold` instead of `==`? I think `==` should be sufficient here as we already know the result of `task.getPIDMode()`, i.e. it's `"task"` instead of `"Task"` or `"tAsk"`, this is more like a contract between Agent and ACS.